controllers/sriovoperatorconfig_controller.go

```diff
@@ -78,6 +78,14 @@ func (r *SriovOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.
 		Name: constants.DEFAULT_CONFIG_NAME, Namespace: namespace}, defaultConfig)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			// Check if we only have one node
+			nodeList := &corev1.NodeList{}
+			err = r.List(context.TODO(), nodeList)
+			if err != nil {
+				return reconcile.Result{}, err
+			}
+
+			disableDrain := len(nodeList.Items) == 1
```

Comment:
We should check that the the controlPlaneTopology = SingleReplica

Reply:
The `machineconfiguration.openshift.io/controlPlaneTopology: SingleReplica` is a MCO label. I want this feature to work also for plain k8s